Dockerfile.clr.boringssl.envoy

```diff
@@ -0,0 +1,39 @@
+FROM clearlinux:base as builder
+
+env HOME /root
+RUN rm -rf /run/lock/clrtrust.lock
+RUN clrtrust generate
+RUN swupd update || swupd update
+RUN swupd bundle-add os-core-dev python2-basic kvm-host
+
+RUN curl -LO https://github.com/bazelbuild/bazel/releases/download/1.1.0/bazel-1.1.0-installer-linux-x86_64.sh
+RUN chmod +x bazel-1.1.0-installer-linux-x86_64.sh
+RUN ./bazel-1.1.0-installer-linux-x86_64.sh --user
```

Comment:
I've asked a bazel update in Clear with the idea that we'd get to use bazel from swupd.

Reply:
Tagging https://github.com/clearlinux/distribution/issues/1507 for tracking.